addons/im_livechat/static/src/components/composer/composer_tests.js

```diff
@@ -0,0 +1,82 @@
+odoo.define('im_livechat/static/src/components/composer/composer_tests.js', function (require) {
+'use strict';
+
+const components = {
+    Composer: require('mail/static/src/components/composer/composer.js'),
+};
+const {
+    afterEach: utilsAfterEach,
+    afterNextRender,
+    beforeEach: utilsBeforeEach,
+    start: utilsStart,
```

Comment:
avoid renaming

Reply:
have to, because test uses own `start` (sets `this.env` and `this.widget`)

Reply:
There's no conflict, one `start` is module scope and the other is written on `this`.
\+ comment also applies to before/after each.